trio/_core/tests/test_run.py

```diff
@@ -2222,8 +2225,10 @@ async def do_a_cancel():
         await do_a_cancel()
         await do_a_cancel()
 
-        async with _core.open_nursery() as nursery:
-            nursery.start_soon(do_a_cancel)
+        with pytest.raises(ValueError):
+            async with _core.open_nursery() as nursery:
+                nursery.start_soon(do_a_cancel)
+                nursery.start_soon(raise_ValueError)
```

Comment:
I think a comment is warranted here to note that we're covering leaks in MultiError by raising this parallel exceptions

Reply:
Additionally, I tried to fully cross-reference all the `del`s and tests.